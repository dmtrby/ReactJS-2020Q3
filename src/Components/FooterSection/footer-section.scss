.footer {
  background: $dark-gray;
  margin-top: auto;
}