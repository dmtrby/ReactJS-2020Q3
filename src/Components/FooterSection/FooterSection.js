import React from 'react';

import Logotype from '../Logotype';

import './footer-section.scss';

const FooterSection = () => (
  <footer className="footer padding-top-1 padding-bottom-1">
    <div className="container">
      <div className="row center-xs">
        <div className="col-xs">
          <div className="row center-xs">
            <Logotype />
          </div>
        </div>
      </div>
    </div>
  </footer>

);

export default FooterSection;
