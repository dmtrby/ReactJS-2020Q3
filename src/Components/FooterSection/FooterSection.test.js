import React from 'react';
import FooterSection from './FooterSection';
import renderer from 'react-test-renderer';

test('FooterSection matches the Snapshot', () => {
  const component = renderer.create(<FooterSection />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
