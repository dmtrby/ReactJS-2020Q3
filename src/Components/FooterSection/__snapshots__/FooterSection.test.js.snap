// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSection matches the Snapshot 1`] = `
<footer
  className="footer padding-top-1 padding-bottom-1"
>
  <div
    className="container"
  >
    <div
      className="row center-xs"
    >
      <div
        className="col-xs"
      >
        <div
          className="row center-xs"
        >
          <span
            className="logotype"
          >
            <strong>
              netflix
            </strong>
            roulette
          </span>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
